src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const strWithStyles = sourceString.split(';');
```

Comment:
The variable name `strWithStyles` could be more descriptive. Consider a name that reflects the content of the variable, such as `styleRules` or `cssDeclarations`.